alcs-frontend/src/app/shared/details-header/details-header.component.html

```diff
@@ -47,9 +47,9 @@ <h5>{{ _application.fileNumber }} ({{ _application.applicant }})</h5>
       </div>
     </div>
     <div>
-      <ng-container *ngIf="!_application.hasOwnProperty('decisionDate')">
+      <ng-container *ngIf="isTimeTrackable && !_application.hasOwnProperty('decisionDate')">
```

Comment:
I'm confused, but it looks like the isTimeTrackable is always false, correct?

Reply:
Removed isTimeTrackable